themes/default/compositions.cjs

```diff
@@ -274,9 +274,11 @@ module.exports = (config, theme) => ({
   '.animated-radio': {
     position: 'relative',
     cursor: 'pointer',
-    fontSize: '1.5em',
-    height: '1em',
-    width: '1em',
+    // fontSize: '1.5em',
+    // replacing `1em` with `24px`
+    // using `em` results in misaligned circles due to half-pixel (e.g. 22.5px) grid alignment issue
```

Comment:
When using `em`, the browser may calculate the width and height to be a non-integer value (e.g. 22.5 pixels). That can lead to misalignment as these items fall onto the pixel grid. Using a round, even number instead, like `24px` makes it easier to create a predictable alignment.

Reply:
Good question. `fontSize` was the attribute driving the base value that informed the `em` unit value in `height` and `width`, which was the underlying issue. Switching to absolute values removed the need for declaring a `fontSize`.